lib/process_settings/file_monitor.rb

```diff
@@ -24,6 +25,26 @@ def initialize(file_path, logger:, environment: nil)
       start_internal(enable_listen_thread?(environment))
     end
 
+    def start_watchdog_thread(file_path)
+      @watchdog_thread and raise ArgumentError, "watchdog thread already running!"
+      @watchdog_thread = Thread.new do
+        watchdog = ProcessSettings::Watchdog.new(file_path)
```

Comment:
I don't think you should need `Processettings::` here since you're already n  that namespace.